apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_connector.erl

```diff
@@ -394,6 +461,8 @@ combine_status(Statuses) ->
             end;
         [Status | _] ->
             Status;
+        [] when ExpectedNoClientids ->
```

Comment:
if a node has no client ID assigned, should probably raise an error so admins know ?

Reply:
I assumed the case where an user wants to configure a certain bridge with only one clientid: in that case, I think it would be a false alarm.  That's why I treated this as normal.

What is not normal is for the whole cluster to have no cilentids assigned, and this is checked in the schema.

It doesn't try to check that there's at least one real, live node with a clientid...  There'll be unrecoverable errors on nodes that receive messages without any clientids, though.

Reply:
if there is only one client ID, but more than one node, the user should be alerted.
e.g. a deployment started with one node, later added another node to the cluster, they will immediately get a warning instead of waiting for the messages to trigger unrecoverable errors.